services/venice-server/src/main/java/com/linkedin/venice/listener/ReadQuotaEnforcementHandler.java

```diff
@@ -100,9 +115,10 @@ public static int getRcu(RouterRequest request) {
       case SINGLE_GET:
         return 1;
       case MULTI_GET:
-        return request.getKeyCount();
+      case MULTI_GET_STREAMING:
```

Comment:
Hmm so previously STREAMING request  will return Int.MAX?

Reply:
Right... it looks like the default is to reject quota for unknown request types since it may not know how to evaluate its cost properly and we forgot to update here when we introduced new request types like `STREAMING`. Let me know if we should change the default.